.github/workflows/gt4py-tox.yml

```diff
@@ -45,4 +50,4 @@ jobs:
         pyversion_no_dot=${pyversion_no_dot//./}
         pyversion_no_dot=${pyversion_no_dot:0:2}
         pip install tox clang-format
```

Comment:
Minor issue unrelated to this PR: I realized that installing `clang-format` does not make sense since tox will create a new venv to run the tests. Most likely the correct way would be adding `clang-format` to the `extras` section in tox.ini

Reply:
Ok, let's do separate.